test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs

```diff
@@ -169,39 +176,54 @@ public void cache_nativeonlyassets()
                 .CopyTestAsset("UnmanagedCache")
                 .WithSource();
 
-            var rid = RuntimeEnvironment.GetRuntimeIdentifier();
-            string libPrefix = "";
-            string arch = rid.Substring(rid.LastIndexOf("-") + 1);
-            string runtimerid = "win7-" + arch;
-
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                libPrefix = "lib";
-                runtimerid = rid;
-            }
-
             ComposeCache cacheCommand = new ComposeCache(Stage0MSBuild, simpleDependenciesAsset.TestRoot);
-           
-            var tfm = "netcoreapp1.0";
+
             var OutputFolder = Path.Combine(simpleDependenciesAsset.TestRoot, "outdir");
+            var WorkingDir = Path.Combine(simpleDependenciesAsset.TestRoot, "composedir");
             cacheCommand
-                .Execute($"/p:RuntimeIdentifier={runtimerid}", $"/p:TargetFramework={tfm}", $"/p:ComposeDir={OutputFolder}", $"/p:DoNotDecorateComposeDir=true")
+                .Execute($"/p:RuntimeIdentifier={runtimerid}", $"/p:TargetFramework={tfm}", $"/p:ComposeWorkingDir={WorkingDir}", $"/p:ComposeDir={OutputFolder}", $"/p:DoNotDecorateComposeDir=true")
                 .Should()
                 .Pass();
 
             DirectoryInfo cacheDirectory = new DirectoryInfo(OutputFolder);
 
-           
-
-
             List<string> files_on_disk = new List<string> {
+               "artifact.xml",
                $"runtime.{runtimerid}.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/{runtimerid}/native/{libPrefix}coredistools{Constants.DynamicLibSuffix}",
                $"runtime.{runtimerid}.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/{runtimerid}/native/coredistools.h"
                };
 
             cacheDirectory.Should().OnlyHaveFiles(files_on_disk);
+        }
+
+        [Fact]
+        public void multifile()
```

Comment:
We've been using the pattern where the test method name describes what the test does and expects.  That way we can easily tell what the test is supposed to do.